llvm/test/DebugInfo/Symbolize/XCOFF/xcoff.test

```diff
@@ -0,0 +1,60 @@
+
+RUN: llvm-symbolizer --obj=%p/Inputs/xcoff-dwarf.o 'DATA 0x60' \
+RUN:    'DATA 0x61' 'DATA 0x64' 'DATA 0X68' 'DATA 0x90' 'DATA 0x94' \
+RUN:    'DATA 0X98' | FileCheck %s
```

Comment:
Nit: if you need to break the command over multiple lines, it's better for the next command to be on the next line.